org.openmw.OpenMW.yaml

```diff
@@ -116,12 +116,24 @@ modules:
       - examples
     build-options:
       env:
-        CXXFLAGS: "-std=gnu++98"
+        CXXFLAGS: "-std=gnu++98 -O2"
```

Comment:
```yaml
  build-options:
    cxxflags: -std=gnu++98
```

(Note you don't need to add -O2, its in default flags)

Reply:
Just copied the exact changes from the shared module, but I agree that the `-O2` felt a bit out of place

Reply:
Ah, the shared-module has a lot of extra complexity becuese it has to support 1.6 and 18.08, like the conditional patch command.

Reply:
Why exactly aren't you using the shared module?